ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -336,8 +350,15 @@ private void updateSubviewClipStatus(View subview) {
 
     // do fast check whether intersect state changed
     sHelperRect.set(subview.getLeft(), subview.getTop(), subview.getRight(), subview.getBottom());
-    boolean intersects = mClippingRect
-        .intersects(sHelperRect.left, sHelperRect.top, sHelperRect.right, sHelperRect.bottom);
+    Matrix matrix = subview.getMatrix();
+    if (!matrix.isIdentity()) {
```

Comment:
Yeah, that sounds reasonable if you can get it to work. The best other solution I can think of is just not clipping transformed views no matter what, but I'm worried some developer is going to translate images (e.g. SoundCloud's android app as you scroll the main screen) and have a bad time.